content/may2015.md

```diff
@@ -54,12 +54,12 @@ We encourage you to explicitly request one of these types via the `Accept` heade
 
 ## Root Endpoint
 
-The FHIR URL takes the form `fhir.sandboxcernerpowerchart.com/may2015/open/:ehr_source_id/:resource`.
+The FHIR URL takes the form `fhir.sandboxcernerpowerchart.com/may2015/:ehr_source_id/:resource`.
 
 ### Open Endpoint
 
-The presence of `/open` in the path allows developers to experiment with the service without requiring
-authentication.
+The `fhir-open.sandboxcernerpowerchart.com` domain allows developers to experiment with the service without requiring
+authentication. Use `fhir.sandboxcernerpowerchart.com` domain to access the authenticated endpoint.
```

Comment:
Small grammatical comment.  Either insert "the" after "Use" (i.e. Use the ... domain) or remove "domain" (i.e. Use ... to access)